ui/src/album/AlbumGridView.js

```diff
@@ -74,43 +108,32 @@ const Cover = withContentRect('bounds')(
 const AlbumGridTile = ({ showArtist, record, basePath }) => {
   const isDesktop = useMediaQuery((theme) => theme.breakpoints.up('md'))
   const classes = useStyles()
-  const [visible, setVisible] = useState(false)
 
   return (
-    <div
-      onMouseMove={() => {
-        setVisible(true)
-      }}
-      onMouseLeave={() => {
-        setVisible(false)
-      }}
-    >
-      <Link to={linkToRecord(basePath, record.id, 'show')}>
+    <div>
+      <Link
+        className={classes.link}
+        to={linkToRecord(basePath, record.id, 'show')}
+      >
         <Cover album={record} />
-        {(!isDesktop || visible) && (
-          <GridListTileBar
-            className={classes.tileBar}
-            title={record.name}
-            subtitle={
-              <div className={classes.albumArtistName}>
-                {showArtist ? (
-                  <ArtistLinkField
-                    record={record}
-                    className={classes.artistLink}
-                  />
-                ) : (
-                  <RangeField
-                    record={record}
-                    source={'year'}
-                    sortBy={'maxYear'}
-                    sortByOrder={'DESC'}
-                  />
-                )}
-              </div>
-            }
-            actionIcon={<AlbumContextMenu record={record} color={'white'} />}
-          />
-        )}
+        <GridListTileBar
+          className={isDesktop ? classes.tileBar : classes.tileBarMobile}
+          subtitle={
+            <PlayButton
+              record={record}
+              className={classes.playButton}
+              size="small"
+            />
+          }
+          actionIcon={<AlbumContextMenu record={record} color={'white'} />}
+        />
+      </Link>
+      <Link
+        className={classes.albumLlink}
+        to={linkToRecord(basePath, record.id, 'show')}
+      >
+        <div className={classes.albumName}>{record.name}</div>
```

Comment:
You should be using `<Typography>` instead of `<div>`, to make the color of the font automatic based on the theme:
<img width="284" alt="Screen Shot 2020-10-23 at 9 12 25 AM" src="https://user-images.githubusercontent.com/331353/97008074-5b476e00-1510-11eb-87d3-1a0ec2659236.png">

Also, see my comment bellow about the color of the `<PlayButton>`